Remotion/ObjectBinding/Web/Globalization/BocList.de.resx

```diff
@@ -171,4 +171,7 @@
   <data name="Remotion.ObjectBinding.Web.UI.Controls.BocList.ListMenuHeading" xml:space="preserve">
     <value>Optionen</value>
   </data>
+  <data name="Remotion.ObjectBinding.Web.UI.Controls.BocList.RemainingValidationFailureText" xml:space="preserve">
+      <value>Auf einer anderen Seite sind {0} Fehler aufgetreten.</value>
```

Comment:
deciding to go with "at least one" is something we should check with our downstream customers. doc team are only helping us find nice wording, not deciding what information is relevant. If it turns out that we don't need to know how many errors there are, we can go with "at least one", otherwise we need two separate resources. Lets talk with Martin tomorrow, and possibly Andreas.